src/components/DefinitionEntry.js

```diff
@@ -71,16 +71,28 @@ export default class DefinitionEntry extends React.Component {
 
   renderHeadline(definition) {
     const { namespace, name, revision } = definition.coordinates
+    const locationUrl = this.getLocationURL(definition)
+    const revisionUrl = this.getRevisionUrl(definition)
     const namespaceText = namespace ? namespace + '/' : ''
     const sourceUrl = this.getSourceUrl(definition)
-    let revisionText = <span>&nbsp;&nbsp;&nbsp;{revision}</span>
+    let revisionText = (
+      <span>
+        &nbsp;&nbsp;&nbsp;
+        <a href={revisionUrl} target="_blank">
```

Comment:
what happens if there is no `revisionUrl`

Reply:
If there is no revisionUrl the revision without a hyperlink. If there is no revision then nothing is shown.